### PR TITLE
Specify node version

### PR DESCRIPTION
**Pull request by bonjourmauko**
Specify `node` version (`lts/boron`), so people can properly install and run the website in their computers.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+lts/boron
```
Comment (sandcha) [inline .nvmrc]: > Improve discoverability by adding it to `package.json`, `engines` section? (not tested ; [example](https://medium.com/@adambisek/how-to-check-minimum-required-node-js-version-4a78a8855a0f))

Comment (maukoquiroga) [reply .nvmrc]: OK

File: README.md [hunk 2]
```diff
@@ -2,37 +2,37 @@
...
 ## [EN] Introduction
 
-[OpenFisca](http://openfisca.org/en) is a versatile microsimulation free software.  
+[OpenFisca](http://openfisca.org/en) is a versatile microsimulation free software.
 This is the source code of the [web site](http://fr.openfisca.org) for the community working on the french tax and benefit system.
 
 ## Install
 
-> This project uses [Next.js](https://github.com/zeit/next.js) and [React](https://reactjs.org).  
+> This project uses [Node.js (lts/boron)](https://nodejs.org), [Next.js](https://github.com/zeit/next.js) and [React](https://reactjs.org).
```
[the remark this answers is not in the document]
Comment (maukoquiroga) [reply README.md]: I removed the .nvmrc file, and added it to engines.

[the remark this answers is not in the document]
Comment (sandcha) [reply README.md]: Great for `.nvmrc`. 👍 
Nevertheless, if the README states a version name, by updating node version in `package.json` we will have to think about updating the README and it's easy to forget.   
So, is stating node *version* in this README necessary? 😕

**Review by sandcha — APPROVED** [review]
One note but nothing blocking. 🙂

**Comment by maukoquiroga** [comment]
LTS/Boron is in maintenance this month, and only security upgrades will be pushed. It is safe to say 6.14.x will be the last LTS Boron minor.

**Comment by sandcha** [comment]
As discussed, we decided to keep it in `package.json` along with other version numbers and remove it from README 🙂 Thank you @maukoquiroga

**Pull request merged**